examples/viewer_lib/ui/segmentation/segment_rendering_ui.py

```diff
@@ -0,0 +1,105 @@
+from dataclasses import dataclass, field
+
+from trame_server.utils.typed_state import TypedState
+from trame_vuetify.widgets.vuetify3 import (
+    Template,
+    VBtn,
+    VBtnToggle,
+    VCard,
+    VCardItem,
+    VCardText,
+    VSlider,
+)
+
+from trame_slicer.segmentation import SegmentationOpacityEnum
+
+from ..utils import ControlButton, FlexContainer, Text
+
+
+@dataclass
+class SegmentRenderingState:
+    rendering_mode: SegmentationOpacityEnum
+    opacity_2d: float = 0.5
+    opacity_3d: float = 1.0
+    show_3d: bool = False
+    rendering_options: list[str] = field(default_factory=lambda: ["filled", "outlined"])
```

Comment:
Why is this in the state? If you only allow two opacity modes, this could be defined directly in the UI Ctor instead and not forwarded in the state at all.